netbox/virtualization/migrations/0039_convert_disk_size.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 5.0.6 on 2024-06-06 17:46
+
+from django.db import migrations
+from django.db.models import F
+
+
+def convert_disk_size(apps, schema_editor):
+    VirtualMachine = apps.get_model('virtualization', 'VirtualMachine')
+    VirtualMachine.objects.update(disk=F('disk') * 1000)
```

Comment:
Can we exclude records with null values, just for efficiency?
```suggestion
    VirtualMachine.objects.filter(disk__isnull=False).update(disk=F('disk') * 1000)
```